go.mod

```diff
@@ -3,11 +3,11 @@ module github.com/Finschia/wasmd
 go 1.20
 
 require (
-	github.com/Finschia/finschia-sdk v0.47.0-rc6
-	github.com/Finschia/ibc-go/v3 v3.3.3
-	github.com/Finschia/ostracon v1.0.10-0.20230417090415-bc3f5693b6a1
+	github.com/Finschia/finschia-sdk v0.47.1-0.20230725074611-f8840edecbaa
+	github.com/Finschia/ostracon v1.1.0
 	github.com/Finschia/wasmvm v1.1.1-0.11.2.0.20230418093236-ce70a3856778
 	github.com/cosmos/iavl v0.19.4
+	github.com/cosmos/ibc-go/v4 v4.3.1
```

Comment:
Why use cosmos/ibc-go instead of finschia/ibc-go?

Reply:
This is because finschia/ibc-go/v4 is named `cosmos/ibc-go/v4` https://github.com/Finschia/ibc-go/blob/main/go.mod#L3. (finschia/ibc-go/v3 was named `finschia/fbc-go/v3`)
So, we cannot declare it as "finschia/ibc-go/v4" and we need to use replacing feature.